src/components/motebehov/MotebehovKvittering.tsx

```diff
@@ -84,43 +85,36 @@ const composePersonSvarText = (
     ? " - " + tilLesbarDatoMedArUtenManedNavn(svarOpprettetDato)
     : undefined;
 
-  return [personIngress, personNavn, svarResultat, opprettetDato]
-    .filter(Boolean)
-    .join(" ");
+  return (
+    <span>
+      <b>{personIngress}</b> {personNavn}, {svarResultat} {opprettetDato}
+    </span>
+  );
 };
 
 interface MotebehovKvitteringInnholdProps {
   deltakerOnskerMote?: boolean;
   ikonAltTekst: string;
   motebehov?: MotebehovDTO;
-  tekst: string;
+  tekst: ReactElement;
+  topPadding: boolean;
```

Comment:
Smak og behag, men kanskje bedre med `topPadding?: PaddingSize;` her også i steden for boolsk prop.

Reply:
Sant, kan endre :)